### DISCUSSION TITLE
where's the config file?

### DISCUSSION THREAD

**Discussion by joshgura** [oobabooga/text-generation-webui#6320] [Q&A]
slogging my way through the errors, i've finally gotten a model to load. it's not responding. want to try boot parameters. but just for future reference I need to know where to write the boot parameters for oobabooga. old thread said "webui.py" but i'm not finding such a file.  Where's the place to write the boot parameters? in stable diffusion frontends, it's in webui-user.sh for example.

**Comment by Touch-Night** [comment]
If you mean command flags, edit CMD_FLAGS.txt
